src/componentes/LogOut/LogOut.jsx

```
@@ -1,9 +1,5 @@
-import React from "react";
-
-const LogOut = (props)=>{
-    return(
-        <div onClick={() => props.out()}>Sign out</div>
-        )
-}
-
-export {LogOut}
\ No newline at end of file
+const LogOut = (props) => {
+	return <div onClick={() => props.out()}>Sign out</div>;
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)